### ISSUE TITLE
How do you choose the best checkpoint to eval this model on test dataset?

### ISSUE THREAD

**Issue by imzhangyd** [HaojunYu1998/UltraDet#5]
Thanks for your contributions. I'd like to learn how you choose the best checkpoint to eval this model on test dataset.

**Comment by imzhangyd** [comment]
Hi @xjtulyc . Are you using the parameters from the last saved checkpoint, or are you using the parameters from a checkpoint with the highest metric on the validation set to eval its performance on test split? Could you provide more details? 

Also, could you explain how the pretraining parameters file 'ultradet.pth' was obtained? What were the training and validation datasets used?

Thanks.

**Comment by xjtulyc** [comment]
The weights of 'ultradet' are obtained from the last trained checkpoint. The hyperparameters were tuned based on the validation set. 'ultradet.pth' is the model obtained after training.

**Comment by imzhangyd** [comment]
Thanks for your reply. I noticed in your paper that you mentioned, 'We perform 4-fold cross-validation and report the mean values and standard errors on the test set to mitigate fluctuations.' I understand that you performed four-fold cross-validation on the training data, meaning you trained the model four times and obtained four models. I would like to know which fold the training result 'ultradet.pth' corresponds to.
